scripts/create-secrets/prompts/list-with-transformer.js

```diff
@@ -0,0 +1,94 @@
+const List = require("inquirer/lib/prompts/list");
+const chalk = require("chalk");
+const figures = require("figures");
+const cliCursor = require("cli-cursor");
+
+class listWithTransformer extends List {
```

Comment:
If this is a class it should start with a capital - i.e. `ListWithTransformer`

Reply:
updated this~